### PR TITLE
llava 1.6 finetuning

### PR DESCRIPTION
**Pull request by Sato-Daichi**
The original LLaVA repo does not support to fine-tune LLaVA 1.6. So, I made this PR to do it.

This code is from https://github.com/haotian-liu/LLaVA/pull/1347

### CODE REVIEW

**Comment by feiliya333** [comment]
Thanks for your great PR.

I encountered a bug that the loss is zero from the beginning of the training process, after modifying the code according to your commit. I am finetuning llava-vicuna-7b, (already set image_aspect_ratio to anyres, mm_patch_merge_type to spatial_unpad, chat template to be llava_v1). 
Have you encountered the bug? Or maybe I missed something?

**Comment by treeaaa** [comment]
Hi,I used your modification and it's running now
Thank you so much
